test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/services/chat-states/ChatStates.spec.js

```diff
@@ -513,6 +513,30 @@ describe("Chat States", () => {
             expect(container.transcript.addSystemMsg).toHaveBeenCalledWith({msg: "Adviser exited chat"});
         });
 
+        it("does not report HMRC loses connection ", () => {
```

Comment:
in this test and the one below I think we should spy on the new function and assert it is called once